sklearn/metrics/_ranking.py

```diff
@@ -807,6 +816,20 @@ def _binary_clf_curve(y_true, y_score, pos_label=None, sample_weight=None):
 
     thresholds : ndarray of shape (n_thresholds,)
         Decreasing score values.
+
+    Examples
+    -------
```

Comment:
```suggestion
    Examples
    --------
```

Reply:
Since this is just right after the return, it might just the parser that fail to get the right error here.

Reply:
this works but I dont know why

Reply:
In rst header underlining just needs to be at least as long as the header, see: https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#sections

The parser did give the wrong error, which we could look into though.

Reply:
yea